src/com/google/enterprise/adaptor/DocumentHandler.java

```diff
@@ -1252,8 +1267,12 @@ private void transformMetadata() {
       }
       crawlOnce = Boolean.parseBoolean(params.get(KEY_CRAWL_ONCE));
       lock = Boolean.parseBoolean(params.get(KEY_LOCK));
-      // TODO: make constants for this growing set of valid keys
-      considerNotSending(params.get(KEY_TRANSMISSION_DECISION), docId);
+      String transmissionDecision
+          = params.get(KEY_FORCED_TRANSMISSION_DECISION);
```

Comment:
So filters can override an already forced decision.  My only remaining thoughts are whether forced is the right adjective.  For example would coerced be preferable because it seems less final?